packages/angular-mcp-server/src/lib/tools/ds/report-violations/group-violations.tool.ts

```diff
@@ -0,0 +1,243 @@
+import { createHandler } from '../shared/utils/handler-helpers.js';
```

Comment:
I suggest to delete comments as they are not giving more context than method names that are already there